HealthcareSystem/PatientWebApp/Controllers/FeedbackController.cs

```diff
@@ -30,6 +32,8 @@ public FeedbackController(IFeedbackService feedbackService)
         /// </summary>
         /// <param name="id">id of the wanted object</param>
         /// <returns>if alright returns code 200(Ok), if not 404(not found)</returns>
+        /// 
+        [AllowAnonymous]
         [HttpGet("{id}")]
         public IActionResult GetFeedbackById(int id)
         {
```

Comment:
Does this only return the published or all? Only admin should see the unpublished ones. (Although, is this endpoint ever used anyway? I can't remember any use case that requires you to fetch only one feedback.)